cmds/deploy.go

```diff
@@ -160,7 +166,7 @@ func NewCmdDeploy(f *cmdutil.Factory) *cobra.Command {
 
 						tmpl.Parameters = append(tmpl.Parameters, tapi.Parameter{
 							Name:  "DOMAIN",
-							Value: domain,
+							Value: apidomain,
```

Comment:
Correct: one for domain that can be used for the route creation & oauth redirect; one for the API server address.